src/firefly/html/lc.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <link rel="icon" type="image/x-icon" href="images/fftools-logo-16x16.png">
+    <meta name="apple-mobile-web-app-capable" content="yes" />
+    <title>Light Curve Viewer</title>
+
+    <script>
+        window.firefly = {
+            app: {
+                template: 'LightCurveViewer',
```

Comment:
Yes, FireflyViewer is the default.  It's documented via jsdoc in FFEntryPoint.app.